components/rmrk/Gallery/Item/Sharing.vue

```diff
@@ -138,6 +150,10 @@ export default class Sharing extends Vue {
     this.$buefy.toast.open(message)
   }
 
+  // get currentGalleryItemImage(): { image: string; name: string } {
+  //   return this.$store.getters['history/getCurrentlyViewedItem'] || {}
+  // }
```

Comment:
Why not put the logic here so that we don't need to write `$store.getters['history/getCurrentlyViewedItem']` many times.